<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaseListWindow</class>
 <widget class="QMainWindow" name="CaseListWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>病歷查詢</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QTableWidget" name="tableWidget_patient_list">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>病歷號碼</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>病患姓名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>性別</string>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>出生日期</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>身份證號</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>國籍</string>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>保險類別</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>優待類別</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>初診日期</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>聯絡電話</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>行動電話</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>電子郵件</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>聯絡地址</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>檔案</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>編輯</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="palette">
    <palette>
     <active/>
     <inactive/>
     <disabled/>
    </palette>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_requery"/>
   <addaction name="separator"/>
   <addaction name="action_open_record"/>
   <addaction name="action_delete_record"/>
   <addaction name="separator"/>
   <addaction name="action_export_patient_list"/>
   <addaction name="separator"/>
   <addaction name="action_close"/>
  </widget>
  <action name="action_open_record">
   <property name="icon">
    <iconset>
     <normaloff>../icons/document-open.svg</normaloff>../icons/document-open.svg</iconset>
   </property>
   <property name="text">
    <string>調閱資料</string>
   </property>
   <property name="toolTip">
    <string>調閱資料</string>
   </property>
  </action>
  <action name="action_delete_record">
   <property name="icon">
    <iconset>
     <normaloff>../icons/user-trash.png</normaloff>../icons/user-trash.png</iconset>
   </property>
   <property name="text">
    <string>資料刪除</string>
   </property>
  </action>
  <action name="action_export_patient_list">
   <property name="icon">
    <iconset>
     <normaloff>../icons/document-export.svg</normaloff>../icons/document-export.svg</iconset>
   </property>
   <property name="text">
    <string>匯出名單</string>
   </property>
  </action>
  <action name="action_requery">
   <property name="icon">
    <iconset>
     <normaloff>../icons/reload.svg</normaloff>../icons/reload.svg</iconset>
   </property>
   <property name="text">
    <string>重新查詢</string>
   </property>
   <property name="toolTip">
    <string>重新查詢病歷</string>
   </property>
  </action>
  <action name="action_close">
   <property name="icon">
    <iconset>
     <normaloff>../icons/application-exit.svg</normaloff>../icons/application-exit.svg</iconset>
   </property>
   <property name="text">
    <string>關閉視窗</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>pushButtonQueryClicked()</slot>
  <slot>pushButtonRequeryClicked()</slot>
  <slot>tableWidgetCellDoubleClicked()</slot>
  <slot>actionOpenCaseTriggered()</slot>
  <slot>actionDeleteCaseTriggered()</slot>
  <slot>actionPrintPresTriggered()</slot>
 </slots>
</ui>
